plans/all.fmf

```diff
@@ -0,0 +1,46 @@
+adjust:
+    when: initiator is not defined or initiator != packit
+    discover+:
+        dist-git-source: true
+
+discover+:
+    how: fmf
+execute:
+    how: tmt
+
+/basic_check:
+    summary: Run basic checks
+    discover+:
+        test: /test/basic_check
+
+/podman_e2e_test_upstream:
+    summary: Run SELinux specific Podman e2e tests on upstream PRs
+    discover+:
+        test: /test/podman_e2e_test_upstream
+    adjust:
+        enabled: false
+        when: initiator is not defined or initiator != packit
```

Comment:
This can be moved to `test` section. You should test it though, some `context` are not available at that stage, but I believe `initiator` will always be available.